app/src/components/buttons/multiActionButton/multiActionButton.scss

```diff
@@ -0,0 +1,149 @@
+.multi-action-button {
+  position: relative;
+  display: inline-block;
+  height: 30px;
+  box-sizing: border-box;
+  cursor: pointer;
+
+  &.disabled {
+    opacity: 0.4;
+    cursor: default;
+    pointer-events: none;
+  }
+  &.opened {
+    .toggle-icon {
+      transform: rotate(180deg);
+    }
+    .menu {
+      display: block;
+    }
+  }
+  &.color-booger {
+    .button-part {
+      background-color: $COLOR--booger;
+
+      &:hover {
+        background-color: darken($COLOR--booger, 15%);
```

Comment:
Are there special colors for the disabled buttons in the palette?
https://zpl.io/bokl9Kv - I see here some colors for disabled state, but they are not from our palette ( https://zpl.io/a3qYxMd ). Let's clarify this

Reply:
I just used colors like in `BigButton` component.
So I need to clarify it with Designer